docker/Dockerfile

```diff
@@ -1,68 +1,66 @@
 # Copyright 2015 Telefónica Investigación y Desarrollo, S.A.U
 #
-# This file is part of lightweightM2M-iotagent
+# This file is part of the IoT Agent for Lightweight M2M (iotagent-lwm2m) component
 #
-# lightweightM2M-iotagent is free software: you can redistribute it and/or
+# iotagent-lwm2m is free software: you can redistribute it and/or
 # modify it under the terms of the GNU Affero General Public License as
 # published by the Free Software Foundation, either version 3 of the License,
 # or (at your option) any later version.
 #
-# lightweightM2M-iotagent is distributed in the hope that it will be useful,
+# iotagent-lwm2m is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
 # See the GNU Affero General Public License for more details.
 #
 # You should have received a copy of the GNU Affero General Public
-# License along with lightweightM2M-iotagent.
+# License along with iotagent-lwm2m.
 # If not, see http://www.gnu.org/licenses/.
```

Comment:
Please change:

iotagent-lwm2m -> lightweightm2m-iotagent

as that is supposed to be the name of the repo.

Reply:
Fixed 4e3aea9